scene/resources/default_theme/default_theme.cpp

```diff
@@ -440,6 +440,7 @@ void fill_default_theme(Ref<Theme> &theme, const Ref<Font> &default_font, const
 	theme->set_color("completion_font_color", "TextEdit", Color::html("aaaaaa"));
 	theme->set_color("font_color", "TextEdit", control_font_color);
 	theme->set_color("font_color_selected", "TextEdit", Color(0, 0, 0));
+	theme->set_color("font_color_read_only", "TextEdit", Color(control_font_color.r, control_font_color.g, control_font_color.b, 0.5f));
```

Comment:
The property is named `readonly` in one word, so it's probably best if the color follows the same spelling.